gradle.properties

```diff
@@ -1,3 +1,5 @@
 kotlin.code.style=official
-kotlinVersion=1.3.70
+kotlinVersion=1.3.72
```

Comment:
It's better to store versions in a separate file that was proposed previously. Purpose of `gradle.properties` file is different.